go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/onflow/flow-go-sdk
 go 1.13
 
 require (
-	cloud.google.com/go v0.97.0 // indirect
 	cloud.google.com/go/kms v1.0.0
-	github.com/ethereum/go-ethereum v1.9.9
-	github.com/onflow/cadence v0.20.1
-	github.com/onflow/flow-go/crypto v0.24.3
-	github.com/onflow/flow/protobuf/go/flow v0.2.2
+	github.com/ethereum/go-ethereum v1.9.13
+	github.com/onflow/cadence v0.21.2
+	github.com/onflow/flow-go v0.24.6
+	github.com/onflow/flow-go/crypto v0.21.4-0.20211125190211-7b31c986316e
```

Comment:
It looks like this downgrades the `flow-go/crypto` version. Is that intentional?

Reply:
We should not downgrade the crypto version, if there is an issue with the newest crypto version, we can work together to fix it.